constants.py

```diff
@@ -16,5 +16,13 @@
 RAW_FOLDER = Path(DATA_ROOT_FOLDER, "raw")
 GROUND_REFERENCE_FOLDER = Path(RAW_FOLDER, "ground-reference")
 
+# Path to parent remote folder with all missions
+ALL_MISSIONS_REMOTE_FOLDER = "js2s3:ofo-public/drone/missions_01"
+
 # Intermediate
 PHOTOGRAMMETRY_FOLDER = Path(DATA_ROOT_FOLDER, "intermediate", "photogrammetry")
+PREPROCESSING_FOLDER = Path(DATA_ROOT_FOLDER, "intermediate", "preprocessing")
```

Comment:
We should probably at some point all plan together how we want to organize all these data folders